packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -248,7 +248,6 @@ class Stdio {
 
   Stream<List<int>> get stdin => io.stdin;
 
-  @visibleForTesting
```

Comment:
This was used in `lib/src/test/` which was incorrectly though reasonably detected as allowed via `visibleForTesting`.  When I moved it into `lib/src/commands/test.dart` then it complained.  Just remove this annotation.